mqtt-impl/src/main/java/io/streamnative/oidc/broker/common/package-info.java

```diff
@@ -0,0 +1,15 @@
+/**
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+// Copy from sn-pulsar-plugins, only used to compile stage
+package io.streamnative.oidc.broker.common;
```

Comment:
Not sure if it conflicts with the package on `sn-pulsar-plugins`, if conflicts may
 be we need to exclude this package when building